cmd/policy-actions.go

```diff
@@ -0,0 +1,246 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/evertras/bubble-table/table"
+	"github.com/opentdf/otdfctl/pkg/cli"
+	"github.com/opentdf/otdfctl/pkg/man"
+	"github.com/opentdf/platform/protocol/go/policy"
+	"github.com/spf13/cobra"
+)
+
+func policyGetAction(cmd *cobra.Command, args []string) {
+	c := cli.New(cmd, args)
+	h := NewHandler(c)
+	defer h.Close()
+
+	id := c.Flags.GetOptionalID("id")
+	name := c.Flags.GetOptionalString("name")
+
+	if id == "" && name == "" {
+		cli.ExitWithError("Either 'id' or 'name' must be provided", nil)
+	}
+
+	action, err := h.GetAction(cmd.Context(), id, name)
```

Comment:
Shouldn't the handlers for these commands just use `h.ctx` like most of the other policy object commands? Although, it is confusing because I see passed-in contexts like you are using in kas grants handlers, as well as `context.Background()` used in the resource mapping handlers...

Reply:
@c-r33d is going to be updating all the existing handlers to take in an argument populated by `cmd.Context()` as we have an open issue to refactor our use of struct contexts, which are considered a code smell. https://github.com/opentdf/otdfctl/issues/364